src/mumble/main.cpp

```diff
@@ -378,6 +393,25 @@ int main(int argc, char **argv) {
 		}
 	}
 
+	// Populate extra translation dirs (directories in which Mumble shall look for
+	// translation files that can overwrite the bundled one if present). The order
+	// in which they are added to this list determines the priority (first match
+	// will win).
+	// Start with the directory the Mumble executable lives in.
+	extraTranslationDirs.append(a.applicationDirPath());
+	// Next add AppData directories
+	for (const QString &currentConfigPath : QStandardPaths::standardLocations(QStandardPaths::AppDataLocation)) {
+		extraTranslationDirs.append(QDir::cleanPath(currentConfigPath));
+	}
+	// Next add directories in the config locations
+	for (const QString &currentConfigPath : QStandardPaths::standardLocations(QStandardPaths::ConfigLocation)) {
+		extraTranslationDirs.append(QDir::cleanPath(currentConfigPath + QDir::separator() + "MumbleTranslations"));
+	}
+	// Finally add a dir in the home directory
+	for (const QString &currentHomePath : QStandardPaths::standardLocations(QStandardPaths::HomeLocation)) {
+		extraTranslationDirs.append(QDir::cleanPath(currentHomePath + QDir::separator() + "MumbleTranslations"));
+	}
+
```

Comment:
Maybe we could move the translation handling logic into a separate class rather than making main.cpp even longer?

Reply:
Yeah that would probably be a good idea :+1: